src/kbmod/standardizers/butler_standardizer.py

```diff
@@ -0,0 +1,337 @@
+"""Class for standardizing Data Products of Vera C. Rubin Science Pipelines
+via the Rubin Data Butler.
+"""
+import importlib
+import uuid
+
+import astropy.nddata as bitmask
+from astropy.wcs import WCS
+import numpy as np
+from scipy.signal import convolve2d
+
+from .standardizer import Standardizer, StandardizerConfig
+from kbmod.search import LayeredImage, RawImage, PSF
+
+
+__all__ = [
+    "ButlerStandardizer",
+    "ButlerStandardizerConfig",
+]
+
+
+def deferred_import(module, name=None):
+    """Imports module/class/function/name as  ``name`` into global modules.
+    If module/class/function/name already exists does nothing.
+
+    Intended for importing a large module or a library only when needed, as to
+    avoid the cost of the import if the functionality depending on the imported
+    library is not being used.
+
+    Used to defer the import of Vera C. Rubin Middleware component only during
+    `ButlerStandardizer` initialization so that we may be able to import KBMOD
+    before the heat death of the universe.
+
+    Parameters
+    ----------
+    module : `str`
+        Fully qualified name of the module/submodule/class/function/name.
+    name : `str` or `None`, optional
+        Name the target is imported as.
+
+    Raises
+    ------
+    ImportError :
+        Target is not reachable. Ensure package is
+        installed and visible int the environment.
+    """
+    if name is None:
+        name = module
+    if globals().get(name, False):
+        return
+    try:
+        globals()[name] = importlib.import_module(module)
+    except ImportError as e:
+        raise ImportError(f"Module {module} or name {name} not found.") from e
+
+
+class ButlerStandardizerConfig(StandardizerConfig):
+    do_mask = True
+    """Perform masking if ``True``, otherwise return an empty mask."""
+
+    do_bitmask = True
+    """Mask ``mask_flags`` from the mask plane in the FITS file."""
+
+    do_threshold = False
+    """Mask all pixels above the given count threshold."""
+
+    grow_mask = True
+    """Grow mask footprint by ``grow_kernel_shape``"""
+
+    brightness_treshold = 10
```

Comment:
The Rubin images are normalized to 0 I think, so the values should be reasonable? But I have no idea. This is copied directly from the old before mask refactoring and were set on probably 1 or 2 example images I had. We can adjust later.

https://github.com/dirac-institute/kbmod/blob/88afe4628570ff8033b0fad533d092fc9fead6a8/src/kbmod/run_search.py#L77

but the 

https://github.com/dirac-institute/kbmod/blob/88afe4628570ff8033b0fad533d092fc9fead6a8/src/kbmod/configuration.py#L61

so this would never execute. I did the same in this PR, I just made it a flag that is `False` by default: 

https://github.com/dirac-institute/kbmod/blob/c00519f20992f9a27294089dfc12c4ef29ccf7fc/src/kbmod/standardizers/butler_standardizer.py#L64

because I rather had multiple values to set than to mix logic with numerical parameters. Should I remove to match the masking refactor?

Reply:
If you would prefer to have a separate logic value and numeric threshold, I'm fine keeping it that way.